plugins/explore/src/components/GroupsExplorerContent/GroupsDiagram.tsx

```diff
@@ -96,7 +96,9 @@ function RenderNode(props: DependencyGraphTypes.RenderNodeProps<any>) {
           alignmentBaseline="baseline"
           style={{ fontWeight: 'bold' }}
         >
-          {props.node.name}
+          {props.node.profile?.displayName
```

Comment:
You can do 
```ts
props.node.profile?.displayName ?? props.node.name
```

Instead rather than repeating the ternary

Reply:
ah you're right, thank you

Reply:
I have modified it can you take a look again @benjdlambert ? And I am not sure about this checklist `Added or updated documentation.` What documentation that I need to update?

Reply:
got it @freben